.github/workflows/ci-conda-workflow.yml

```diff
@@ -204,7 +204,6 @@ jobs:
         source ${CONDA}/etc/profile.d/conda.sh
         conda activate build
         conda install -yq pytest-cov
-        cd $HOME
```

Comment:
I'm concerned about removing this, since I think (it's hard to tell) that this is meant to test the installed version.

Of course, it would be easier if we had #1608 merged as I have not looked at the "setup.py test" code for a long while as it's deprecated.

Reply:
Actually, I had added `cd $HOME`, and then removed it after it broke (because that test runs the setup.py, which is located in the work dir, not HOME). 

If you take a look at the final diff, not the diff between commits, you'll see where I added `cd $HOME`.

background: trying to debug why the tests get different results, and it looks like because the tests are run from different locations, causing different coverage reports. it would seem some are running `.../work/sherpa/sherpa/` and others are working from `.../work/sherpa/sherpa/sherpa/` (it breaks down to .../work/sherpa<org name>/sherpa<repo name>/sherpa<the normal dir in the repo>/`). I hadn't rebased/squashed while doing it, so you're seeing those diffs.

now that I've fixed that, the coverage reports something closer to expected, 83.11%.